.github/workflows/copilot_deploy.yml

```diff
@@ -67,18 +67,22 @@ jobs:
     runs-on: ubuntu-latest
     outputs:
       matrix: ${{ steps.copilot_env_matrix.outputs.env_list }}
+      pre_matrix: ${{ steps.copilot_env_matrix.outputs.pre_deploy_list }}
+      post_matrix: ${{ steps.copilot_env_matrix.outputs.post_deploy_list }}
     steps:
       - id: copilot_env_matrix
         run: |
           if [ "${{ inputs.environment }}" != "" ]; then
             echo "env_list=[\"${{ inputs.environment }}\"]" >> $GITHUB_OUTPUT
           elif [ "${{ github.ref }}" == 'refs/heads/main' ]; then
-            echo "env_list=[\"dev\", \"test\", \"uat\", \"production\"]" >> $GITHUB_OUTPUT
+            echo "pre_deploy_list=[\"dev\", \"test\"]" >> $GITHUB_OUTPUT
+            echo "post_deploy_list=[\"uat\", \"production\"]" >> $GITHUB_OUTPUT
           else
-            echo "env_list=[\"dev\", \"test\"]" >> $GITHUB_OUTPUT
+            echo "pre_deploy_list=[\"dev\", \"test\"]" >> $GITHUB_OUTPUT
```

Comment:
Does post_matrix not need to be set to []?

Reply:
That's covered by line 215:

https://github.com/communitiesuk/funding-service-design-notification/pull/164/files#diff-491c3b45d6d2d7b65cd8d91331f0f1d86f73428bbf0d232671ff0e44be5eb85bR215